plover/registry.py

```diff
@@ -54,6 +55,17 @@ def update(self):
             entrypoint_type = 'plover.%s' % plugin_type
             for entrypoint in pkg_resources.iter_entry_points(entrypoint_type):
                 self.register_plugin(plugin_type, entrypoint)
+            if sys.platform.startswith('darwin'):
```

Comment:
Wouldn't it be better to move this block out of the loop?

And perhaps we can stick with 'darwin' or 'mac' or 'macos' instead of 'osx', since that name is being retired. I'm undecided on which I like.